assets/js/base/context/providers/cart-checkout/checkout-processor.js

```diff
@@ -33,6 +33,12 @@ import { useStoreNotices } from '../../hooks/use-store-notices';
  * Subscribes to checkout context and triggers processing via the API.
  */
 const CheckoutProcessor = () => {
+	const controller = useRef(
+		typeof AbortController === 'undefined'
+			? undefined
+			: new AbortController()
```

Comment:
We dropped support for IE11 so in theory it should be okay to just assume this is safe to use? What are your thoughts on that?